package.json

```diff
@@ -15,7 +15,8 @@
 		"nohoist": [
 			"**/config",
 			"**/testarmada-magellan",
-			"**/testarmada-magellan-*"
+			"**/testarmada-magellan-*",
+			"calypso/@wordpress/interface"
```

Comment:
This is required because https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/style.scss#L1 does `@import "../../interface/src/style.scss";`, which assumes `@wordpress/edit-post` and `@wordpress/interface` will always be at the same level